assets/js/util/svg.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Utility functions for SVGs.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { renderToString } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import ChangeArrow from '../components/ChangeArrow';
+
+/**
+ * Gets the up/down SVG arrow for Google Chart tooltips.
+ *
+ * @since n.e.x.t
+ *
+ * @param {number}  difference  The difference which can be positive or negative.
+ * @param {boolean} invertColor Whether we need to reverse the +/- colors.
+ * @return {string} SVG for the +/- difference.
+ */
+export const getChartDifferenceArrow = ( difference, invertColor = false ) => {
```

Comment:
Let's make the second argument into an options object to avoid passing a boolean as a positional argument. This is helps with readability because it isn't obvious what that refers to otherwise. I don't think we're actually using that argument yet so it should just be a simple signature change.

Reply:
Updated @aaemnnosttv